public/app/core/services/echo/backends/analytics/ApplicationInsightsBackend.ts

```diff
@@ -0,0 +1,46 @@
+import $ from 'jquery';
+import { EchoBackend, EchoEventType, isInteractionEvent, isPageviewEvent, PageviewEchoEvent } from '@grafana/runtime';
+
+export interface ApplicationInsightsBackendOptions {
+  instrumentationKey: string;
+}
+
+export class ApplicationInsightsBackend implements EchoBackend<PageviewEchoEvent, ApplicationInsightsBackendOptions> {
+  supportedEvents = [EchoEventType.Pageview];
+
+  constructor(public options: ApplicationInsightsBackendOptions) {
+    $.ajax({
+      url: 'https://js.monitor.azure.com/scripts/b/ai.2.min.js',
+      dataType: 'script',
+      cache: true,
+    }).done(function () {
+      var applicationInsightsOpts = {
+        config: {
+          instrumentationKey: options.instrumentationKey,
```

Comment:
It seems there is recommendation to use connectionString instead of instrumentationKey. 
https://docs.microsoft.com/en-us/azure/azure-monitor/app/javascript#adding-the-javascript-sdk

Reply:
I'll fix!